.all-contributorsrc

```diff
@@ -11,12 +11,13 @@
       "avatar_url": "https://avatars1.githubusercontent.com/u/25856189?v=4",
```

Comment:
We should use the AllContributors bot to manage this file, by mentioning it in comments:

https://allcontributors.org/docs/en/bot/usage

If we manually change that file, unexpected things might happen. Let's leave these changes out of the pull request.